app/routes/application.vue

```diff
@@ -0,0 +1,121 @@
+<template lang="html">
+  <div class="">
+
+
+    <nav class="nav">
+      <div class="nav-left">
+        <router-link to="{ name: 'index' }" class="nav-item is-brand">Puppies</router-link>
```

Comment:
The `to` property should be `:to` when using objects and not just pure string URLs.